### ISSUE TITLE
xSPSecureStoreServiceApp should use PSCredential instead of username and password parameters

### ISSUE THREAD

**Issue by KarolKaczmarek** [PowerShell/xSharePoint#87]
This bug is a result of PSScriptAnalyzer scan:
```
RuleName                                                                         Severity              FileName                                                             Line                  Message                                              
--------                                                                                 --------                    --------                                                                    ----  -------                                              
PSAvoidUsingUserNameAndPassWordParams     Error                     MSFT_xSPSecureStoreServiceApp.psm1 122   Function 'Test-TargetResource' has both username and 
password parameters. A credential parameter of type PSCredential should be used.                         
PSAvoidUsingUserNameAndPassWordParams    Error                     MSFT_xSPSecureStoreServiceApp.psm1 51    Function 'Set-TargetResource' has both username and  
password parameters. A credential parameter of type PSCredential should be used.                                                                                 
PSAvoidUsingUserNameAndPassWordParams     Error                      MSFT__xSPSecureStoreServiceApp.psm1 1     Function 'Set-TargetResource' has both username and  
password parameters. A credential parameter of type PSCredential should be used.                         

```

We should have single parameter credential of type PSCredential instead of using both username and password parameters.

**Comment by BrianFarnhill** [comment]
The fix for this is in #97 and I will close this issue off once it has passed review and is merged in to the dev branch

**Comment by BrianFarnhill** [comment]
PR #97 is now merged in to dev and this issue has been resolved.
